### PR TITLE
Split visual style page in subpages

### PR DESCRIPTION
**Pull request by Volker-E**
Splitting visual style page in subpages to be easier
digestable and amendable.
Also adding agreed-upon contents in Illustrations and add some
other contents too.

### CODE REVIEW

File: visual-style_colors.html [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+									</div>
+								</li>
+							</ol>
+						</div>
+
+						<h3>Additional colors</h3>
+						<p>Some use cases, such as charts and data visualization, may need a broader color palette. Make sure to aim for level AA contrast (4.5:1) when extending the default palette. Also try to test how they are perceived at different color vision deficiency conditions.</p>
+					</section>
+
+					<section id="typography">
```
Comment (prtksxna) [inline visual-style_colors.html]: Accidentally left this in?

File: visual-style.html [hunk 2]
```diff
@@ -43,12 +43,12 @@ <h1 class="site__title"><a href="./"><span class="site__logo"></span>Wikimedia S
 						<li class="nav__item"><a href="design-principles.html">Design principles</a></li>
 						<li class="nav__item is-on"><a href="visual-style.html">Visual style</a>
 							<ul class="nav__sub-items">
-								<li class="nav__sub-item"><a href="#principles">Principles</a></li>
-								<li class="nav__sub-item"><a href="#colors">Colors</a></li>
-								<li class="nav__sub-item"><a href="#typography">Typography</a></li>
-								<li class="nav__sub-item"><a href="#icons">Icons</a></li>
-								<li class="nav__sub-item"><a href="#imagery">Imagery</a></li>
-								<li class="nav__sub-item"><a href="#illustrations">Illustrations</a></li>
+								<li class="nav__sub-item is-on"><a href="visual-style.html">Principles</a></li>
```
Comment (prtksxna) [inline visual-style.html]: Github is not letting me leave a comment on Line 95, so I'll leave one here. I think you've accidentally left a few sections on this page.

**Comment by Volker-E** [comment]
@prtksxna https://phabricator.wikimedia.org/T160598

File: visual-style_colors.html [hunk 3]
```diff
@@ -0,0 +1,361 @@
...
+<!--[if lte IE 8 ]><html lang="en" class="js-off lte-ie8"><![endif]-->
+<!--[if IE 9 ]>    <html lang="en" class="js-off ie9"><![endif]-->
+<!--[if (gt IE 9)|!(IE)]><!-->
+<html lang="en" class="js-off">
+<!--<![endif]-->
+	<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no">
+	<link rel="stylesheet" href="css/wmui-style-guide.min.css">
+	<title>WikimediaUI Style Guide – Visual Style: Colors</title>
```
Comment (juliengirault) [inline visual-style_colors.html]: This one has `Visual Style: Colors` but the others don't follow the same pattern

**Pull request merged**